src/common/timeout-registry.ts

```
@@ -0,0 +1,27 @@
+  }
+
+  cancel(timerId: TimerId | null | undefined): void {
+    if (timerId === null || timerId === undefined) {
+      return;
+    }
+    scheduler.cancelTimeout(timerId);
+    this.timeouts.delete(timerId);
+  }
+
+  cancelAll(): void {
+    this.timeouts.forEach((timerId) => scheduler.cancelTimeout(timerId));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)